server/web/edit/index.hbs

```diff
@@ -22,9 +22,9 @@ This edit will create a new revision.
         >
     </div>
     <div>
-      <label for="author-email">Email </label>
+      <label for="authorEmail">Email </label>
```

Comment:
these form values need to match `home/index.hbs` (maybe could abstract) for the form submission to work